docs/prometheus.md

```diff
@@ -0,0 +1,21 @@
+## syzkaller metrics
+
+syz-manager metrics are exposed at the URI : http://<syz-manager-http-uri>/metrics
```

Comment:
The part between <> is not shown. Check out how it looks after rendering markdown.
Also: no space before `:` and `.` at the end of the sentence.